samples/Nancy.Demo.Authentication.Stateless/AuthModule.cs

```diff
@@ -1,27 +1,28 @@
 ﻿namespace Nancy.Demo.Authentication.Stateless
 {
-    public class AuthModule : LegacyNancyModule
+    public class AuthModule : NancyModule
     {
         public AuthModule() : base("/auth/")
         {
             //the Post["/login"] method is used mainly to fetch the api key for subsequent calls
-            Post["/"] = x =>
-                {
-                    string apiKey = UserDatabase.ValidateUser((string) this.Request.Form.Username,
-                                                              (string) this.Request.Form.Password);
+            Post("/", args =>
+            {

```

Comment:
I know these are demo project's but the file above this has the brace open on the same line, then this opens on the next line!

I'm becoming the stylecop now.